Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -795,8 +794,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Checks if the user is part of built-in privileged Active Directory
-        groups.
+      description: Checks if the user is part of built-in privileged Active Directory groups.
```

Comment:
Checks whether the user is part of any built-in privileged Active Directory groups.